config/internal_users.yml

```diff
@@ -61,3 +61,8 @@ snapshotrestore:
   backend_roles:
   - "snapshotrestore"
   description: "Demo snapshotrestore user, using external role mapping"
+
+new-user:
```

Comment:
I don't think we should add a demo user here. If this is used for an integration test, can this be moved to a test file?